src/components/Header/index.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { Hyperlink } from '@edx/paragon';
+import { Dropdown, Hyperlink } from '@edx/paragon';
```

Comment:
The dropdown arrow doesn't animate like it does on Studio. Is that just because paragon doesn't support that?

Reply:
Yeah, that's what it looks like. I'm actually a bit surprised it does do that in Studio.